android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.CHANGE_WIFI_MULTICAST_STATE" />
```

Comment:
nit: add comment about react-native-zeroconf requiring these permissions

Reply:
Good idea. Done in 91eed19545ebeb40bb57cbc24890dd9bffdd9843.